include/mbedtls/aes.h

```diff
@@ -37,6 +37,7 @@
 #define MBEDTLS_AES_DECRYPT     0
 
 #define MBEDTLS_ERR_AES_INVALID_KEY_LENGTH                -0x0020  /**< Invalid key length. */
+#define MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE               -0x0021  /**< Feature not available, e.g. unsupported AES key size. */
```

Comment:
When you add an error code, you need to run `scripts/generate_errors.pl`, which you've done, and update a comment in `include/mbedtls/error.h`, which you haven't done. (This is not at all obvious, sorry, we should write a KB article about it.)

I made a copy-pasto in my previous comment, sorry about that. #1178 already uses 0x0021. @andresag01 could one of you please switch to 0x0023 (ideally by amending your last commit), add it to `error.h` (as in https://github.com/ARMmbed/mbedtls/pull/1178/files#diff-c52ac35237c20a39a4621bdcaf1501ce), and make sure there's a CI run with the latest commit?

Reply:
Alright, I did not know what you wanted to do because the link was not correct in your previous comment. Also, I did not add the other range to error.h because I thought it unnecessary. In any case I will change it to your preferred way